util/SerializeUniverse.cpp

```diff
@@ -250,9 +251,12 @@ void Fleet::serialize(Archive& ar, const unsigned int version)
         & BOOST_SERIALIZATION_NVP(m_next_system)
         & BOOST_SERIALIZATION_NVP(m_aggressive)
         & BOOST_SERIALIZATION_NVP(m_ordered_given_to_empire_id)
-        & BOOST_SERIALIZATION_NVP(m_travel_route)
-        & BOOST_SERIALIZATION_NVP(m_travel_distance)
-        & BOOST_SERIALIZATION_NVP(m_arrived_this_turn)
+        & BOOST_SERIALIZATION_NVP(m_travel_route);
+    if (version < 3) {
+        double dummy_travel_distance;
+        ar & boost::serialization::make_nvp("m_travel_distance", dummy_travel_distance);
```

Comment:
could just call the dummy variable `m_travel_distance` and use `BOOST_SERIALIZATION_NVP`

Reply:
I did not change this.

Calling the variable `m_travel_distance`, implies for a casual reader that there is a member variable due to our naming conventions.  There are reasonable scenarios where a developer code remove/modify the line of code because they know the `m_travel_distance` is not a member variable and break the deserialization.

The proposed line of code can only be read as serialize/deserialize a double in the archive named `m_travel_distance` into a variable called `dummy_travel_distance`.  The name `dummy_X` implies that it is not used.